dashboard/src/components/DeploymentFormBody/DeploymentFormBody.v2.tsx

```diff
@@ -145,7 +145,7 @@ function DeploymentFormBody({
       </div>
       <div className="deployment-form-control-buttons">
         <CdsButton status="primary" type="submit">
-          <CdsIcon shape="deploy" inverse={true} /> Deploy v{chartVersion}
+          <CdsIcon shape="deploy" inverse={true} /> Deploy v{selected.version?.attributes.version}
```

Comment:
In what cases is `selected.version` undefined?

Reply:
here it's never undefined because it's checked before but typescript did't detect it, rewriting to make it more clear